.github/workflows/lint.yml

```diff
@@ -0,0 +1,38 @@
+name: Lint
+
+on: [push, pull_request]
```

Comment:
This could bother us every commit, WDYT about just running this when a PR is merged or a commit is made on master?
This way we'll do that as a cleanup only on the last commit of the pr (the merge)

Reply:
I think no. This linter is running for contributors, not maintainers. You, maintainers, are shared properly setting, there is no reason commits with wrong indentation.
However, we need to notice immediately that the maintainer's commits have been mixed in as well.
That's why I think this trigger is appropriate.

Reply:
This linter will run for pushes on master by maintainers as well, but my concern is more if I have a PR and push a commit, the lint will run, when I try to push another commit, it won't work, I'll need to pull the last commit, that's why I don't like CI committing on PRs until it's finished.

Running it on master pushes or PR merges would work for both contributors and maintainers

Reply:
This job is not pushing anything. It's only check if there is diff with what the auto indentation and current code, and fail if there's different

Reply:
>  if I have a PR and push a commit, the lint will run,

yes

> when I try to push another commit, it won't work

but, really?
I think it should run too.

Reply:
Oh sorry, I thought the CI was pushing, failing when is not linted correctly is enough for me :)

Reply:
> This job is not pushing anything. It's only check if there is diff with what the auto indentation and current code, and fail if there's different

Yes. That is CI. You just don't merge PR if any CI fail.

Reply:
Yes, is just that I already worked on projects that the linter pushes to the branch every commit and I thought this a PR that was doing that, sorry for that

Reply:
No need to say sorry. Let me know if it doesn't work as I say :)